src/model/Note.ts

```diff
@@ -810,35 +811,29 @@ export class Note {
                             this.bendType = BendType.BendRelease;
                         } else if (!isContinuedBend && origin.value > 0) {
                             this.bendType = BendType.PrebendBend;
-                            points.splice(2, 1);
-                            points.splice(1, 1);
+                            points.splice(1, isFourPointsBend ? 2 : 1);
```

Comment:
nit: Can you move this index into a variable and reuse it. Having so many copy pastes feels unclean.

Reply:
reverted